README.md

```diff
@@ -6,7 +6,7 @@ SonataAdminBundle - The missing Symfony2 Admin Generator
 Demo
 ----
 
-The demo website can be found in http://demo.sonata-project.org/admin/dashboard
+The demo website can be found in http://demo.sonata-project.org/admin/dashboard    
```

Comment:
What do you mean?

Can you simply add a line separation between this and the "Username and password is:"?

This is more clear and will avoid problem with auto trailling whitespaces remover on some IDE.